### ISSUE TITLE
Media key support

### ISSUE THREAD

**Issue by NoShotz** [AurousGroup/Feature-Tracker#9]
Allow play/pause next/previous media keys to control the program

**Comment by jppgibbs** [comment]
Yes! This so much. I won't be able to switch to using it until it has media button support.
